bridge/client/app/app.module.ts

```diff
@@ -1,72 +1,74 @@
-import {NgModule} from '@angular/core';
```

Comment:
I guess this is the
> Format/cleanup code according to tslint-rules
stuff :+1:

It's a bit scary, as I think it will create conflicts when merging other PRs that are in progress. I'm pinging @ermin-muratovic on this, so he knows upfront and can later deal with the merge conflicts.

Reply:
true... it is also because I didn't switch of IntelliJ's automatic formatting. For the imports there are not tslint rules yet, so I could change my formatting settings without red lines reappearing 😄 

this PR is a lot smaller than Ermin's so it would be easier to revert here I guess

Reply:
Thank you for the cleanup. I merged it properly into the other PRs.